wiki/rats-mischief.md

```diff
@@ -6,8 +6,110 @@ slug: rats-mischief
 <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 [![img](https://img.shields.io/discord/292744693803122688?color=informational&label=Ladysnake&logo=Discord)](https://ladysnake.glitch.me)[![img](http://cf.way2muchnoise.eu/full_rats-rats_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](http://cf.way2muchnoise.eu/versions/minecraft_rats-rats_latest.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](https://img.shields.io/github/last-commit/ladysnake/rats-mischief)](https://github.com/ladysnake/rats-mischief/commits/main)
 
+
+# Rat's Mischief
+
+[![img](https://img.shields.io/discord/292744693803122688?color=informational&label=Ladysnake&logo=Discord)](https://ladysnake.glitch.me)[![img](http://cf.way2muchnoise.eu/full_rats-mischief_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](http://cf.way2muchnoise.eu/versions/minecraft_rats-mischief_latest.svg)](https://www.curseforge.com/minecraft/mc-mods/rats-mischief)[![img](https://img.shields.io/github/last-commit/ladysnake/rats-mischief)](https://github.com/ladysnake/rats-mischief/commits/main)
+
 **Rat's Mischief** (also known as "Rat's Rats" or just "Mischief") is a Minecraft mod introducing a new mob, the rat, as well as mechanics revolving around these new possible pets.
 
+[![Showcase Video](https://img.youtube.com/vi/CVZfsPM8Mm4/0.jpg)](https://www.youtube.com/watch?v=CVZfsPM8Mm4)
+
+# Wiki
```

Comment:
There should only be 1 first-level title per page, so all those titles should be one level lower
```suggestion
## Features
```